src/config/webpack.common.js

```diff
@@ -17,16 +17,22 @@ const copies = {
 const { eslintConfig, main: entryPoint } = require(resolveExtensionPath("package.json"));
 const eslintEnabled = eslintConfig || fs.readdirSync(extensionPath).find(f => f.startsWith(".eslintrc"));
 const jsLoaders = [{
-    loader: require.resolve("babel-loader"),
```

Comment:
If I remember it right, this helps when you run zem locally within its own directory. 

How is your local development workflow? If we have a better way to run/test zem locally we can get rid of this.

☝️ We have another `require.resolve` usage in this module.

Reply:
I found several places that we use `require.resolve` and fix it. As far as I understand `require.resolve` import the module and return the file path instead of module itself. In our case this is redundant, since webpack can resolve the module from file name.

We also use it at manifest-builder: `delete require.cache[require.resolve(pkgInfoPath)];`. I'm not sure that we should remove from here also. That's why I didn't remove it.